exporters/jaeger/src/recordable.cc

```diff
@@ -77,7 +77,10 @@ void Recordable::AddEvent(nostd::string_view name,
 void Recordable::SetInstrumentationLibrary(
     const opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary
         &instrumentation_library) noexcept
-{}
+{
+  AddTag("otel.library.name", instrumentation_library.GetName());
+  AddTag("otel.library.version", instrumentation_library.GetVersion());
```

Comment:
LGTM, I am referring naming from here to validate : https://github.com/open-telemetry/opentelemetry-specification/blob/v1.4.0/specification/trace/sdk_exporters/non-otlp.md#instrumentationlibrary ?